cpp/include/cudf/detail/nvtx/ranges.hpp

```diff
@@ -39,7 +39,7 @@ struct libcudf_domain {
  * }
  * ```
  */
-using thread_range = ::nvtx3::domain_thread_range<libcudf_domain>;
+using thread_range = ::nvtx3::scoped_range_in<libcudf_domain>;
```

Comment:
suggestion (minor, non-blocking): Bonus points for renaming `cudf::thread_range` to `cudf::scoped_range`. `thread_range` was the old name that we ultimately renmaed to `scoped_range` instead.

Reply:
Renamed in 7002fc7.